components/apimgt/org.wso2.carbon.apimgt.rest.api.dcr/src/main/java/org/wso2/carbon/apimgt/rest/api/dcr/web/impl/RegistrationServiceImpl.java

```diff
@@ -262,8 +273,9 @@ private OAuthApplicationInfo getExistingApp(String applicationName, boolean saas
 
             appToReturn = this.fromAppDTOToApplicationInfo(consumerAppDTO.getOauthConsumerKey(),
                     consumerAppDTO.getApplicationName(), consumerAppDTO.getCallbackUrl(),
-                    consumerAppDTO.getOauthConsumerSecret(), saasApp, null, consumerAppDTO.getTokenType(), valueMap);
-
+                    consumerAppDTO.getOauthConsumerSecret(), saasApp, MultitenantUtils
```

Comment:
This fails when create oauth app using tenant user ==> https://github.com/wso2/api-manager/issues/2064